.env.example

```diff
@@ -4,4 +4,8 @@ VITE_AWS_COGNITO_IDENTITY_POOL_ID_ASIA=XX-XXXX-X:XXXXXXXX-XXXX-1234-abcd-1234567
 VITE_AWS_REGION_ASIA=XX-XXXX-X
 VITE_AWS_CF_TEMPLATE=https://example.com
 VITE_APPLE_APP_STORE_LINK=https://apps.apple.com/app/amazon-location-demo/id1670477797?trk=18347227-315a-4761-a6a7-07e2ffdf08b6&sc_channel=el
-VITE_GOOGLE_PLAY_STORE_LINK=https://play.google.com/store/apps/details?id=com.aws.amazonlocation&trk=7246fb27-1ed0-4c04-ba03-7af6c090739b&sc_channel=el
\ No newline at end of file
+VITE_GOOGLE_PLAY_STORE_LINK=https://play.google.com/store/apps/details?id=com.aws.amazonlocation&trk=7246fb27-1ed0-4c04-ba03-7af6c090739b&sc_channel=el
+VITE_PINPOINT_REGION=XX-XXXX-X
+VITE_PINPOINT_APPLICATION_ID=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+VITE_PINPOINT_ACCESS_KEY_ID=XXXXXXXXXXXXXXXXXXXX
+VITE_PINPOINT_SECRET_ACCESS_KEY=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```

Comment:
@NihaalAftab-Makeen keeping secret in the env will be a security concern, AppSec will probably escalate this issue

Reply:
Fixed